microcosm_pubsub/caching/resource_cache.py

```diff
@@ -0,0 +1,27 @@
+from microcosm.api import defaults, typed
+from microcosm.config.types import boolean
+from microcosm_pubsub.caching.memcached import MemcachedCache
+
+
+@defaults(
+    enabled=typed(boolean, default_value=True),
```

Comment:
Might be safer to set default value to `False` before we make a release here.. WDYT?

Reply:
Selective updates preferred for big changes like this. Roll out slowly, so rollback isn't painful.

Reply:
Which is to say, yes, false by default.